shimmingtoolbox/dicom_to_nifti.py

```diff
@@ -1,20 +1,98 @@
-# coding: utf-8
+#!usr/bin/env python3
+# -*- coding: utf-8
 
 # TODO: check in unit test if dcm2bids_scaffold is installed, and also check for the required version.
 
+from distutils.dir_util import copy_tree
+import json
+import numpy as np
+import os
+import sys
 import subprocess
+import dcm2bids
+# from dcm2bids.scaffold import scaffold
 
 from shimmingtoolbox import __dir_config_dcm2bids__
 
+# TODO: check in unit test if dcm2bids_scaffold is installed, and also check for the required version.
```

Comment:
Add as an issue instead of a todo. Everything else seems good, i'll approve right after